### ISSUE TITLE
[ios] in lockdown mode the UI becomes unusable

### ISSUE THREAD

**Issue by papamoose** [advplyr/audiobookshelf-app#796]
### Steps to reproduce

1. Settings > Privacy & Security > Lockdown Mode > Turn On Lockdown Mode
2. Phone restarts then launch Audiobookshelf
3. Note the UI text

### Expected behaviour
- UI should look the same as when the phone is NOT in lockdown mode.

### Actual behaviour
- menu items are not hidden inside menus.
- font and size is different
- words used instead of images

### Environment data

Audiobookshelf Version: 0.9.65-beta client 
V2.3.3 server

- [ ] Android App?
- [x] iOS App?

#### iOS Issue
iOS Version: 16.5.1 (c)

iPhone model: 13 Pro

# Images

## lockdown mode on
![image](https://GitHub.com/advplyr/audiobookshelf-app/assets/686499/79f01698-07d5-4639-aec6-64de66c2470e)
![image](https://github.com/advplyr/audiobookshelf-app/assets/686499/dfd70911-d7fb-45f1-89b7-189ad8696cce)
![image](https://github.com/advplyr/audiobookshelf-app/assets/686499/d4351451-1a53-480d-bd2b-ef03cb732c8a)

## lockdown mode off
![image](https://github.com/advplyr/audiobookshelf-app/assets/686499/54d6d026-fa7e-4924-b69e-5633b20dd815)

**Comment by benonymity** [comment]
I don't think this is something that can be easily fixed on the app's side, as Apple seems to be blocking all custom fonts and icon packs in webView, which is an intentional security choice that would be challenging to get around

**Comment by benonymity** [comment]
Probably related to a doubling down of a CORS policy

**Comment by advplyr** [comment]
The fonts are stored locally, no external requests are made for fonts

**Comment by benonymity** [comment]
Gotcha, wouldn't be CORS then. Looks like they're flat-out disabling any sort of SVG font:
> Previewing files in web browsers is also restricted with Lockdown Mode. For instance, JPEG 2000 images and SVG fonts, which are exclusively supported by Safari, are disabled so websites can’t use these formats to target iOS users. PDF previewing for websites is also disabled, as multiple PDF-related exploits have been found in the past.
>[link](https://9to5mac.com/2022/07/25/lockdown-mode-ios-16-restricts-web-browsing/)
